Packs/CommonTypes/IndicatorFields/indicatorfield-region.json

```diff
@@ -24,13 +24,7 @@
     "content": true,
     "group": 2,
     "hidden": false,
-    "associatedTypes": [
-        "IP",
-        "IPv6",
-        "CIDR",
-        "IPv6CIDR",
-        "STIX Threat Actor"
-    ],
+    "associatedTypes": [],
```

Comment:
What do you think with keeping the field for Threat Actor indicator type? For example, a group that operates from Russia, China, etc.'